src/api_authentication_app_skeleton/spec/support/app_visitor.cr

```diff
@@ -0,0 +1,131 @@
+require "http/client"
+
+class AppVisitor
+  include Auth::ActionHelpers
+
+  CSRF_TOKEN = "statictesttoken"
+
+  getter! response
+  @response : HTTP::Client::Response? = nil
+
+  def visit(path : String, as_user : Authentic::PasswordAuthenticatable? = nil, headers : HTTP::Headers? = nil)
```

Comment:
Let's rename this to `get` so it's more consistent with the other HTTP methods and could you rename `as_user` to `as` to match LuckyFlow?

Reply:
Sure, I originally had it as `as` but I was worried that it might be a restricted keyword. If there is no issues then it definitely looks better